cmd/multi-factor-auth/config/api.toml

```diff
@@ -15,11 +15,11 @@ RestApiInterface = ":8080" # The interface `address and port` to which the REST
 
 [APIPackages.guardian]
     Routes = [
-        { Name = "/register", Open = true, Auth = true },
-        { Name = "/sign-message", Open = true, Auth = false },
-        { Name = "/sign-transaction", Open = true, Auth = false },
-        { Name = "/sign-multiple-transactions", Open = true, Auth = false },
-        { Name = "/verify-code", Open = true, Auth = true },
+        { Name = "/register", Open = true, Auth = true , MaxContentLength = 500 },
+        { Name = "/sign-message", Open = true, Auth = false, MaxContentLength = 500 },
+        { Name = "/sign-transaction", Open = true, Auth = false, MaxContentLength = 500 },
+        { Name = "/sign-multiple-transactions", Open = true, Auth = false, MaxContentLength = 500000 },
+        { Name = "/verify-code", Open = true, Auth = true, MaxContentLength = 500 },
```

Comment:
should also add `MaxContentLength = 10` for the all other endpoints, otherwise tcs won't start

Reply:
you are right. But since the other are GET requests and therefore the `ContentLenght` will be 0. Do we still require the `minContentLength`?

Reply:
true.. added new comments with proper fix, no need for default values on get. Thanks for the clarification